### DISCUSSION TITLE
Clone/Duplicate an Issue

### DISCUSSION THREAD

**Discussion by Archdoog** [community/community#8485] [Projects and Issues] [answered]
Hello, we're using issues and projects across three closely related repositories. We would love the ability to simply clone/duplicate an issue, then potentially transfer it to a different repo with the existing transfer.

Thanks!

**Comment by drone1** [comment]
Would also really like this, for splitting a larger issue into smaller issues, or more generally just being able to more quickly create tasks that will populate a particular view & sub-view board column.

**Reply by mehulrajdev** [reply]
Check this, please

(https://github.com/mehulrajdev/Zayn-Malik))

https://github.com/mehulrajdev

**Comment by drone1** [comment]
Hey, are there plans to do this? It is incredibly cumbersome adding tasks to a view at the moment. 

For example, my current view shows tasks for my next milestone. Simple, right? However, after typing the task title I have to click **Open Item**, then _Convert to Issue_, then _Add a milestone_, then _Add an assignee_. Finally, I have a task for my current view, however it took way too many steps.

Adding a simple 'clone' function would save me -- and I'm guessing many people trying -- a lot of time.

I'm sorry to say it but it feels like whatever team is working on this project simply isn't actually using it.

Thanks.

**Comment by fgnievinski** [comment]
related: https://github.com/google/github-issue-mover/issues/133

**Comment by drbilo** [comment]
+ 1

**Comment by wvmitchell** [comment]
Just want to +1 this

I have a series of "template" drafts that I work off of in a project so that I can create similar issues across several different repositories. Unfortunately, the only way that I can actually use them is to manually copy all the markdown into a new issue. If I could just duplicate it, I'd probably save ~1-2 mintues per issue I create for my team. That really adds up.

**Comment by mgslone** [comment]
Would really love to have this.  We have weekly processes that I have to copy/paste into new issues for each week.  I have to copy over the title, description, tags, priority, and story pointing separately, which seems unnecessarily complicated..

**Comment by jdshkolnik** [comment]
Duplicating an existing issue is a necessity.

**Comment by smassoni-usds** [comment]
Would also love to see this. For similar reasons. Given the issue templates don't play nice with the new Projects, it would be nice to be able to easily duplicate issues we keep around as templates vs copying and pasting a bunch of information.

**Comment by Chris-Petty** [comment]
In a large project with many labels for many subcontexts, often I'm churning out issues that all need the almost the same labels, milestone and project. Configuring an issue template for every combination of labels would be really bad though!

A duplicate button that allows me to create a new issue (in editing state) with the same details would help me a lot; description, assignee, labels, milestone, project...

**Comment by MikkelVase01** [comment]
+1 Please fix it

**Comment by darrenmudd** [comment]
Please give us this

**Comment by jmaguero** [comment]
+1, plzzzz! :D

**Comment by mikimer** [comment]
+1

**Comment by nwoottonsky** [comment]
+1

**Comment by jayeshkataria** [comment]
plus one, this is needed please 🙏

**Reply by NJ-G** [reply]
+1

**Comment by fgnievinski** [comment]
[Kamino](https://github.com/gatewayapps/kamino) can be used to clone an issue to another repository, even across different organizations.

**Reply by ghost** [reply]
Hi @fgnievinski.

Very interesting comment, I'll take a look at [Kamino](https://github.com/gatewayapps/kamino). An question: is  [Kamino](https://github.com/gatewayapps/kamino) similar to the [git-bug](https://github.com/MichaelMure/git-bug)? What is the difference between Kamino and git-bug?

If you have time to answer these two questions, I'm happy for any answer.

**Reply by fgnievinski** [reply]
Kamino just clones issues; git-bug has a much much bigger scope.

**Comment by ghost** [comment]
+1, i would love it ❤️

**Comment by drone1** [comment]
Sadly this is not useful if you don't use Chrome or Edge. Anyway IMHO it's
a basic feature that GitHub could quite easily implement.

**Comment by HaoLiangPao** [comment]
It is strange that **views** in github projects can be duplicated but not the **issues**. This feature would be very useful when managing relatively big projects, thanks.

**Comment by sherryumlah** [comment]
+1 please!

**Comment by jeremydaniels-ch** [comment]
+1 !

**Comment by ssyberg** [comment]
+1

**Comment by mmandrus** [comment]
+1

**Comment by vbuxbaum** [comment]
+1

**Comment by agneszitte** [comment]
+1

**Comment by ghost** [comment]
Hello people!

Please GitHub, give us this

**Comment by tri-duong-valuence** [comment]
+1

**Comment by slambrechts** [comment]
We need this for our research

**Comment by benmoranpm** [comment]
Please, this would be really handy.

**Comment by NeoSecundus** [comment]
+1

**Comment by victor-m-jatoba-alb** [comment]
+1

**Comment by StephanieHerr** [comment]
+1

**Comment by kinantid** [comment]
+1 to this - annoying to have to keep reapplying the milestones, iterations, and tags 🥴

**Comment by berdilaovidiu** [comment]
+1

**Comment by cpyle0819** [comment]
I made a little python script for this. It requires the GitHub CLI.
https://gist.github.com/cpyle0819/27758200253ac2480c13cfde1b887e53

**Comment by DeBraid** [comment]
Seems like a no brainer, please make this happen!

**Comment by juliozup** [comment]
Hey, are there plans to do this? PLEASE!!!!!!!

**Comment by benironside** [comment]
+1. This can't be hard

**Comment by MaksimZinovev** [comment]
Come on. We need this. Please.

**Comment by duprasa** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by ingipuff** [comment]
Hi, I'm trying to figure out how to duplicate an issue in my repo and my search led me to here. What is going on with this ticket and who's actually looking at this? There are a number of places that suggests that we do have the capability to create a duplicate from an issue. Is this actually not possible? If so why is there a setting that allows for marking an issue as a duplicate?
![image](https://github.com/community/community/assets/47579889/27219319-7174-4f52-9868-47e47e4e8adc)

**Comment by Gwalior-84** [comment]
+1

**Comment by Gwalior-84** [comment]
> Hi, I'm trying to figure out how to duplicate an issue in my repo and my search led me to here. What is going on with this ticket and who's actually looking at this? There are a number of places that suggests that we do have the capability to create a duplicate from an issue. Is this actually not possible? If so why is there a setting that allows for marking an issue as a duplicate? ![image](https://user-images.githubusercontent.com/47579889/280091414-27219319-7174-4f52-9868-47e47e4e8adc.png)

Marking as duplicate is quite a different thing to cloning an issue.  You would mark an issue as a duplicate when there are two issues, created (hopefully!) by two different people, addressing the same issue.

**Comment by ghost** [comment]
Do we need to get out our pitchforks 🤣.

**Comment by johnfrazieratms** [comment]
I cannot believe this isn't a feature.

**Comment by sigubrat** [comment]
+1 This would be amazing, especially if it could be done across repositories as well.

**Reply by jca002** [reply]
+1

**Reply by SamJaarsma** [reply]
me too: +14 users at C4T

**Comment by cvelez-fixeads** [comment]
+1 
This feature is mandatory for managing a medium / large project in a professional way

**Comment by KPaul404** [comment]
> Hello, we're using issues and projects across three closely related repositories. We would love the ability to simply clone/duplicate an issue, then potentially transfer it to a different repo with the existing transfer.
> 
> Thanks!

To transfer an existing issue to another repo, you can use the "Transfer issue" option under the issue's actions menu. This allows moving an issue to another repo you have write access to, while preserving things like comments, assignments, and labels.

**Comment by KPaul404** [comment]
Also, Standardize your labeling conventions across repos so cloned/transferred issues retain relevant tagging.

**Comment by dev2008** [comment]
Can't believe this isn't a feature already!

**Comment by yuval-lerner-ownera** [comment]
+1

**Comment by LewisFieldMargin** [comment]
+1

**Comment by kahnos** [comment]
+1

**Comment by TJRawlins** [comment]
2024 and this still isn't a feature. Please implement this simple, yet highly demanded request!

**Comment by YervN** [comment]
+1 please add this feature to github

**Comment by jlang-311** [comment]
+1

**Comment by alex-pey** [comment]
+1

**Comment by kf-vikas** [comment]
Hopefully this gets added as a feature soon 🙏🏻

**Comment by ghost** [comment]
I sorted by oldest and compiled all of the information in this thread up until this point.  We need this yesteryear..  Hopefully, I didn't skip something in my haste.

---


# Stats:

- Originally posted on Dec 6th, 2021, but the idea has been around since at least 2017.
- The original post has 185 upvotes
- More that 42 people commented with +1

 
# Clone Feature Request:

- Original:  https://github.com/orgs/community/discussions/8485?sort=top#discussion-3726593

- https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-5213466
  > "A duplicate button that allows me to create a new issue (in editing state) with the same details would help me a lot; description, assignee, labels, milestone, project..."

# Other Feature Requests:

- Splitting tasks/issues into smaller issues
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-2442936
- Clone to another repo  
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-7761337
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-8142408
    - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-8142416

# Inefficient Workflows:

- Copy/Paste and/or Issue templates aren bad workflows
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-4671008
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-5125058
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-5182445
- Project views can be duplicated, but not issues
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-6420513
- Mark as duplicate != Cloning
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-7187139
- Transfer Issue to new repo != Cloning
  - https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-8142408

# "Not quite...  Solutions":

- https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-3721678
- https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-6067542
- https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-7187139

**Comment by cabbruzzese** [comment]
I want to add that this is not so much a feature request as a feature parity issue. It is a relatively fundamental feature of other tracking systems and one of the reasons I moved a company away from github issues in the past in favor of another solution.

**Comment by alexander-sirris** [comment]
+1

**Comment by austinban** [comment]
Here to join to the outcry. Please add this! 

I was searching for this sure that it already existed much to my disappointment.

**Comment by RobbiewOnline** [comment]
+1 my use-case is that I want to have a similar issue for two different platforms implementations (MAUI WinUI and Catalyst desktop integration), with the same checklist inside them both, a copy / clone option would have been ideal, this should copy the title, description, label(s), milestone(s) by default.

**Comment by edshello** [comment]
So first of all I'd like to give a solid +1 to this request, but as long as there is no proper support for this feature we had to come up with something. Our solution is MacOS specific, but I'm sure a similar approach can be taken on other platforms.

What my team ended up doing was using Shortcuts to essentially chain two [GitHub Cli](https://cli.github.com) commands together, with the issue URL in your clipboard as the starting point. We have the shortcut as a widget on our desktop, so duplication of a ticket to another repo now consists of two steps: copy the url, and pressing the shortcut button.

Below is how the main part of the shortcut. You can follow the same pattern for some additional steps using various GH CLI commands (`gh project item-add`, etc).  **Beware: There is most certainly room for sanitation do be done to the input variables, running these gh cli commands blindly is risky!** 
<img width="1225" alt="image" src="https://github.com/community/community/assets/149619443/8bdb356b-85c6-4431-bfac-728804826691">

To keep it as straightforward as possible, we have duplicated the shortcut for various repositories.

<img width="417" alt="image" src="https://github.com/community/community/assets/149619443/a30d91c6-7f82-4bdd-801b-44a64589517d">



Hope it helps! Approach was inspired by https://github.com/orgs/community/discussions/8485?sort=old#discussioncomment-7187139

**Comment by optimiseddesign** [comment]
Come on!! +1

**Comment by miguelsvq** [comment]
+1

I understand that an issue cloning feature could be unnecessary in the public/free version of GitHub, but I can't find a reason why such a feature is not present in the enterprise version for years. Until it's provided by GitHub, I'm using this browser extension: https://github.com/miguelsvq/github_clone_issue.

**Comment by skyglass** [comment]
+1

**Comment by TD-DO** [comment]
+1

**Comment by qubozik** [comment]
+1

**Comment by andrew-rj** [comment]
+1

**Comment by Nominae** [comment]
+1

**Comment by jlannoy** [comment]
+1

**Comment by pjmalandrino** [comment]
+1

**Reply by ronlipke** [reply]
+1 🙏

**Comment by Menchik** [comment]
+1

**Comment by quynhtran97** [comment]
+1

**Comment by bfct** [comment]
+1

**Comment by backwardstruck** [comment]
+1

**Comment by readyon-reza** [comment]
+1

**Comment by dashworth** [comment]
+1 
We moved from tracking tickets/issues in Jira to Github and the only feature I miss is this one.

**Comment by charles-nava** [comment]
+1

**Comment by MrJustinB** [comment]
+1

**Comment by n-c0de-r** [comment]
+1

**Comment by adrianb-vh** [comment]
+1

**Comment by ec-davis** [comment]
+1

**Comment by arabkin** [comment]
+1

**Comment by MazahirHaroon** [comment]
We have weekly recurring issues in our product and this feature would really save us time.

**Comment by razajafri** [comment]
This is needed. I have an issue that needs to be pretty much duplicated and would love to have this feature. 
+1

**Comment by michaw** [comment]
+1 from me. It's a few of these missing basics that are forcing us to look for another system to track our work.

**Comment by danniejpt** [comment]
+ 1. really need it

**Comment by samuellealb** [comment]
+1

**Comment by madeindex** [comment]
Would love that!

**Comment by augustovoigt** [comment]
+1

**Comment by glitchwizard** [comment]
Yes, need this please

**Comment by fernando-espinosa-clip** [comment]
+1

**Comment by amuller-ar** [comment]
+1

**Comment by juggleking** [comment]
+1

**Comment by griffinfujioka** [comment]
+1

**Comment by bluepete** [comment]
+1

**Comment by raphaeladams** [comment]
+1

**Comment by Blintmester** [comment]
+1

**Comment by fkim7** [comment]
+1

**Comment by saunderst** [comment]
+1

**Comment by AbdulGhani002** [comment]
You can automate the process by setting up a GitHub Action or a script that:
Takes an existing issue.
Copies the title, description, labels, and other relevant details.
Creates a new issue in the target repository.
OR
Use the GitHub API: This can be done via the GitHub API by fetching the issue details from the source repository and then using the API to create a new issue in the target repository.

**Comment by drone1** [comment]
I think the point here is that people are looking for a direct way to do
this from within the actual UI.

**Comment by mperazam05** [comment]
I need this! +1

**Comment by sabit990928** [comment]
Our PM is eager to see it as well.

**Comment by ssarrafan** [comment]
+1 pretty please

**Comment by mnott** [comment]
My [Gitflow](https://github.com/mnott/gitflow) utility has an option to clone an issue.

Examples:

```md
    - Clone issue #245 and empty checkboxes:
        ./gitflow.py clone-issue 245
    - Clone issue #245, keep checkboxes, and replace 'CW35' with 'CW36' in the title:
        ./gitflow.py clone-issue 245 --keep-checkboxes --search "CW35" --replace "CW36"
    - Use regex to replace 'CW' followed by any digits with 'CW36' in the title:
        ./gitflow.py clone-issue 245 --search "CW[0-9]+" --replace "CW36"
    - Clone issue #245 with a completely new title:
        ./gitflow.py clone-issue 245 --title "New Issue Title"
    - Clone issue #245 and remove the original assignees, assign to self, and give a new title:
        ./gitflow.py clone-issue 245 -A -a @me --title "New Issue Title"
        ./gitflow.py clone-issue 245 -A --title "New Issue Title"
    - Clone issue #245 and assign it to specific users:
        ./gitflow.py clone-issue 245 --assignee user1 --assignee user2
```

Under the hoods, it uses the `gh` CLI to do these changes. It can also list issues and search within them. The actual use of the utility is rather a gitflow workflow.

**Comment by cmoulliard** [comment]
+1

**Comment by toniagonzalez** [comment]
+1

**Comment by EvgeniiDC** [comment]
+1

**Comment by MaymoonaAlBoloshi** [comment]
🥹 pls, give .. 

+1

**Comment by svanwart** [comment]
+1

**Comment by workgital** [comment]
This should be easy to implement :)

**Comment by caoimhejyoti** [comment]
pretty please  🙏

**Reply by vamseeachanta** [reply]
pretty please. 
PS: No action or comment for 3 years. So nimble in development.

**Comment by Sukh-P** [comment]
+1 please

**Comment by sabit990928** [comment]
Give this action a try: https://github.com/marketplace/actions/issue-duplicator

It solved my problem. Works perfectly with github projects. You just need to type: /duplicate in the issue comment. Feel free to reach out if you have any questions with the setup.

**Comment by astamato** [comment]
+1

**Comment by cyda89** [comment]
+1, need this as well

**Comment by setoelkahfi** [comment]
I am bumping this. +1

**Comment by ciro-c** [comment]
+1, need this as well

**Comment by Enukio** [comment]
+1

**Comment by KWodarczyk** [comment]
+1 Just like in JIRA you have "Clone++" option

**Comment by UlyssesAlves** [comment]
+1 Sometimes we create issues with more details than necessary and need to break them down into smaller, complementary issues. It would be easier to do so if we could clone/copy the original issue and then edit both the original and the newly copied issue keeping only the relevant details on each one of them.

**Comment by MonirMooghen** [comment]
+1

**Comment by optimiseddesign** [comment]
+1

**Comment by aagosto90** [comment]
+1

**Comment by ketura** [comment]
+1.  When a user submits a bug with multiple sub-issues, it would be nice to be able to clone it to easily create those sub-issues.

**Comment by andyguzmaneth** [comment]
+1

**Comment by jjstafford** [comment]
+1

**Comment by DespuesLeeWallace** [comment]
+1

**Comment by BeccaMG** [comment]
+1

**Comment by drone1** [comment]
++++111111

**Comment by githubtkompare** [comment]
Yes. Please. +1

**Comment by freyandhy** [comment]
Need this feature, please. +1

**Comment by rmistry751** [comment]
+1

**Comment by tbd-dimitri** [comment]
+1

**Comment by sfalk7890** [comment]
+1

**Comment by MarcinVaadin** [comment]
+1

**Comment by drone1** [comment]
Github just added some new features to Projects, like creating an issue
directly from a description, with some new UI, etc, but I'm still not
seeing "clone" or "duplicate" anywhere.

Please, someone tell me I'm missing something.

**Comment by nathan-weinberg** [comment]
This is the year GitHub!

**Comment by thisisjoshford** [comment]
+1 :)

**Comment by chris-tuncap** [comment]
+2

**Comment by JoannaPowell** [comment]
+1

**Comment by jjr5** [comment]
+1

**Comment by ProgrammingByPermutation** [comment]
+1

**Comment by yuval-lerner-ownera** [comment]
+1

**Comment by salevine** [comment]
While I also want this, I went ahead and created a bash script to do just this item:  https://github.com/salevine/GithubIssueCopy
It copies all comments, all labels and has the option to create a relationship from destination back to the source

**Reply by MarianoDev1601** [reply]
Thanks for the script! It really helped me. Currently I'm trying to modify it so so it also copy the "type" and "project" of the issue, but I still can't make it work. Could you possible add that feature to the script?

**Reply by salevine** [reply]
Please have this conversation in that repo under the issues.. That way we can discuss what you need there.

**Comment by jameshaigh54** [comment]
+1

**Comment by jgwind** [comment]
+1

**Comment by aflacake** [comment]
Of course you can **post link issue** in new issue (3 repo). Leave it among your repos.
Interesting features too!

**Comment by maximbaran** [comment]
+1

**Comment by isaaclombardssw** [comment]
\+ 1

**Comment by matthewtobiasz** [comment]
+1 to having the functionality to duplicate an issue

**Comment by calmannix** [comment]
Agreed! We use Github projects, so we don't have to jump between GH or Jira (or whatever). The new sub-issue functionality has been great, but I would love to duplicate an issue (or sub-issue) and place it in another repo

**Comment by josephrlee** [comment]
+1, we used ZenHub before and this was a feature that was extremely useful!

**Comment by smides-nest** [comment]
+1, I looked all over trying to find this feature, which is available in most other issue tracking software.

**Comment by ilazakis** [comment]
+1 on this.

**Comment by philbourg** [comment]
+!

**Comment by uo9qsuf** [comment]
+1

**Comment by Nmloury** [comment]
+1

**Comment by Chaseos** [comment]
+1

**Comment by dimitarprisadnikov** [comment]
+1

**Comment by davidbrucesmith** [comment]
Agree with everyone here:  Project Management is tedious in the present system.

**Reply by jen-ck12** [reply]
+1 Not being able to clone/duplicate an issue is a real bore.

**Comment by sandeep-sen** [comment]
+1 
Gitlab has this feature why is GitHub lagging in feature parity.
https://docs.gitlab.com/user/project/quick_actions/#issues-merge-requests-and-epics

**Comment by jsweby** [comment]
+1

**Comment by vanpham-sw** [comment]
I have issues that have small but important differences, other than that type/labels/domain etc... are the same details. Cloning it then adjust the differences is what I would do in Jira.
And it's boring 🥱  filling the same thing repeatedly

**Comment by Christian-onsai** [comment]
+1

**Comment by khalidchamakh** [comment]
+1

**Comment by AdityaBhutani** [comment]
+1

**Comment by innowave-sb** [comment]
+1. Real silly that there's a 'Make a copy' option for a whole project but not for an individual issue.

**Comment by bevangel** [comment]
+1

**Comment by AnnaOborina** [comment]
+1

**Comment by DanielPettersson81** [comment]
+1

**Comment by coyamat** [comment]
+1

**Comment by Rostislav-Dusnii** [comment]
+1

**Comment by amby** [comment]
+1

**Comment by anandhkb** [comment]
This is really useful feature, please prioritize

**Comment by rahulmistry751** [comment]
+1

**Comment by frmadem** [comment]
+1

**Comment by drone1** [comment]
+1

**Comment by amenity** [comment]
+1

**Comment by krahnzinho** [comment]
+1

**Comment by olve-steen** [comment]
+1

**Comment by mbluestone** [comment]
+100000

**Comment by onnerfors** [comment]
+1

**Comment by bp-brinks** [comment]
We need this !!

**Comment by jwdraut** [comment]
+1

**Comment by oscarvgg** [comment]
+1

**Comment by sonni-dev** [comment]
+1

**Comment by hilehman** [comment]
+1

**Comment by Yorisoft** [comment]
+1

**Comment by ValeriyFromUA** [comment]
+1

**Comment by agenteo** [comment]
It’s frustrating that in 2025 GitHub Projects still doesn't support cloning issues. In many workflows—especially for backlog or sprint planning—it’s common to start with a larger issue and break it down as we refine the backlog. Without cloning, we have to manually copy paste and MANUALLY copy any tags assigned for each.

Error-prone and inefficient.

**Comment by esskar** [comment]
+1

**Comment by labudis** [comment]
Hey all 👋 I'm the Senior Product Manager working on Issues and I have good news that we've started experimenting with a feature that allows users to duplicate issues to accelerate issue creation. Watch this space for further updates!

**Reply by uo9qsuf** [reply]
Great. is there a roadmap for a public beta?

**Reply by labudis** [reply]
We are currently testing a version of this feature internally and hoping to release it to the public in the next few weeks. Stay tuned!

**Comment by augustocesarfmo** [comment]
Precisamos disso urgentemente!

**Comment by jlrobins** [comment]
'Clone issue' that created the new clone with all of the manually settable metadata (project, labels, sprint, milestone, parent, assignees, etc.) but left the primary fields title and description blank would be superior to how Some Other Major Ticket system works. Let the user directly enter those, hit save, and boom, *then* the clone becomes real.

**Reply by devstuff** [reply]
I would take it one step further and have the default description be prefilled from the [ISSUE_TEMPLATE](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) corresponding to the original issue's issue type (this might require also having an issue type defined in the template syntax).

**Comment by BanSimplified567** [comment]
+1

**Comment by labudis** [comment, marked as answer]
We've just [shipped](https://github.blog/changelog/2025-07-17-duplicate-issues-create-from-anywhere-and-more/) the ability to **duplicate an issue to any repository** 🚀 

You can now quickly duplicate any issue 🎉  with the new **Duplicate issue** option, which appears at the bottom of the right issue sidebar.

<video src="https://github.com/user-attachments/assets/e3d5c3f1-5572-4192-8dd5-cf0d4da64b2b" width="100%" controls loop autoplay muted></video>

Here’s how it works:

- Click **Duplicate issue** to open a new issue form prefilled with the original issue’s content. This includes the title, description, assignees, labels, type, projects, and milestone.
- Change the destination repository before creating the duplicated issue, making it easy to copy issues across repositories.
- Edit any details before creating the new issue to fit your workflow.

This feature helps you:

- Reuse issue formats or recurring tasks without manual copy-paste.
- Split large issues into smaller, actionable items.
- Create similar issues across multiple repositories.

Let us know what you think and share your feedback in this discussion 🙌🏻

**Reply by silvioana** [reply]
Hi! Thank you very much for adding this feature. We would also like to be able to duplicate an issue that contains sub-issues. This would involve duplicating both the parent issue and the child issues as well. Is this something you plan to add?

**Reply by labudis** [reply]
@silvioana thank you for this suggestion! I can imagine that being valuable but do you have a particular use case that this would unlock for you? I'd recommend starting a separate discussion in the Community to track that feature request.

**Reply by silvioana** [reply]
Thank you @labudis! Discussion started #167223

**Reply by kimallen** [reply]
This is great- I wish I'd discovered the feature months ago. I'd like if it also duplicated the relationship (which in my one test it did not).

**Comment by RyanEaston-IHC** [comment]
Is this functionality available using the gh cli?

**Comment by miguelsvq** [comment]
Better late than never.
